src/pages/all-accounts.tsx

```diff
@@ -43,9 +44,15 @@ function AllAccountsPage() {
   const styles = useStyles()
   const isSmallScreen = useIsMobile()
   const isWidthMax450 = useMediaQuery("(max-width:450px)")
+  const { t } = useTranslation()
 
-  const switchToMainnetLabel = isSmallScreen ? "Mainnet" : "Switch To Mainnet"
-  const switchToTestnetLabel = isSmallScreen ? "Testnet" : "Switch To Testnet"
+  const switchToMainnetLabel = isSmallScreen
+    ? t("all-accounts-page.switch-to-mainnet-label-short")
```

Comment:
These translation keys look somewhat bulky…

How about `page.all-accounts.actions.switch-to-mainnet.label.{long,short}` and scoping the `useTranslation()` to `pages.all-accounts`?

Reply:
I like your idea of scoping with the `useTranslation` functions but I found some constraints while testing. 

Namespaces don't support nesting with the `.` syntax like the `keys` do.  This means that we could only scope it to `page` and not to `page.all-accounts` which is still an improvement but not as much as it could be.

For example we could use 
```
const { t } = useTranslation("page")
t("all-accounts.actions.switch-to-mainnet.label.short")
```
instead of 
```
const { t } = useTranslation()
t("page:all-accounts.actions.switch-to-mainnet.label.short")
```
with a `translation.json`:
```
{
   "page": {
    "all-accounts": {
      "actions": {
        "switch-to-mainnet": {
          "label": {
            "long": "Switch To Mainnet",
            "short": "Mainnet"
          }
        },
        "switch-to-testnet": {
          "label": {
            "long": "Switch To Testnet",
            "short": "Testnet"
          }
        }
      }
    }
  }
}
```


There is an example in [this](https://www.i18next.com/principles/namespaces#semantic-reasons) section from the docs. Maybe we should consider doing something similar to this?
I'm fine with both but I just wanted to point out some alternatives before we start adding the translations as it probably makes more sense thought about this beforehand.